src/components/commons/RoundedTable.tsx

```
@@ -0,0 +1,353 @@
+import React from 'react';
+import { Check, ChevronDown } from 'lucide-react';
+
+// 공통 스타일 상수
+export const tableStyles = {
```

ESLint

[error] Fast refresh only works when a file only exports components. Use a new file to share constants or functions between components.

(react-refresh/only-export-components)